picard/ui/playertoolbar.py

```diff
@@ -312,6 +321,9 @@ class PlaybackRateButton(QtWidgets.QToolButton):
 
     def __init__(self, parent, playback_rate):
         super().__init__(parent)
+        button_margin = self.style().pixelMetric(QtWidgets.QStyle.PM_ButtonMargin)
+        min_width = get_text_width(self.font(), _('%1.1f ×') % 8.8)
```

Comment:
I'd store '%1.1f ×' in a property:

```python
self.rate_fmt = N_('%1.1f ×')
```

```python
label  = _(self.rate_fmt) % rate
```

Reply:
Fixed as part of https://github.com/metabrainz/picard/pull/1231/commits/164c32ff0131de30f5a97634f1a97e5de157f46c